debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/ReplicationMessage.java

```diff
@@ -27,7 +27,7 @@
      *
      */
     public enum Operation {
-        INSERT, UPDATE, DELETE
+        INSERT, UPDATE, DELETE, DDL
```

Comment:
Not sure if I should call the empty event operation DDL since supposedly things like Materialized views an vacuum will send empty events too. Thoughts?

Reply:
If the empty events are actually empty, then why not call them EMPTY.

Reply:
Good point.

Reply:
`EMPTY` sounds good, alongside with a description of possible uses.